internal/observability/call.go

```diff
@@ -83,6 +83,8 @@ func (c call) EndWithPanic(err error) {
 }
 
 func (c call) endLogs(elapsed time.Duration, err error, isApplicationError bool) {
+	appErrBitWithNoError := isApplicationError && err == nil // ie Thrift exception
```

Comment:
let's be a bit more specific here in the comments. 
```
// true if success or a thrift error
appErrBitWithNoError := isApplicationError && err == nil
```

Reply:
Actually, this case is only for Thrift exceptions, not successes.